nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-audit/src/test/java/org/nuxeo/elasticsearch/TestAuditWithElasticSearch.java

```diff
@@ -269,7 +275,8 @@ public void testSaveAndScroll() throws Exception {
         ObjectMapper mapper = new ObjectMapper();
         for (int i = 0; i < 42; i++) {
             ObjectNode logEntryJson = mapper.createObjectNode();
-            logEntryJson.put("eventId", "idForAuditStorage");
+            logEntryJson.put(LOG_ID, Integer.valueOf(i).longValue());
+            logEntryJson.put(LOG_EVENT_ID, "idForAuditStorage");
```

Comment:
I suggest to define a constant `ID_FOR_AUDIT_STORAGE = "idForAuditStorage";`

Reply:
I will use a local variable instead as it's only use in one test (among 3 classes, but it's a known refactoring to do).